deployments/chart/README.md

```diff
@@ -57,7 +57,7 @@ Here is a brief overview of all the values contained in this chart:
 
 - <a id="arguments"></a>`arguments`: Defines command line arguments for the container.
 
-- <a id="includedResources"></a>`includedResources`: Defines all the resources that the go-kube-downscaler is supposed to be able to handle.
+- <a id="includedResources"></a>`includedResources`: Defines all the resources that the go-kube-downscaler is supposed to be able to handle. This list can not be empty.
```

Comment:
Actually it can, the downscaler just won't do anything. (unless something in the helm chart makes it not work)
The default set in the main.go should be overwritten with an empty string list, so it will just loop through nothing every scan